### DISCUSSION TITLE
Preserve zoom state when moving between splits.

### DISCUSSION THREAD

**Discussion by elamaranae** [ghostty-org/ghostty#3317] [Feature Requests, Ideas] [answered]
This is a personal preference and I am not sure if any one else wants this. But, when having multiple splits and current active split is already zoomed in, I often want to switch to other splits without unzooming.

Is this possible to achieve with a config option?

In tmux, I was using the below to perform switch pane and zoom based on current zoomed flag,

```
bind -n C-h if-shell -F "#{window_zoomed_flag}" 'select-pane -L ; resize-pane -Z' 'select-pane -L'
```

**Comment by mitchellh** [comment, marked as answer]
#3484 sorry I answered that one first. Please move to there.
